### DISCUSSION TITLE
Π2019021/Παραδοτεο 6

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/sw#1535] [Show and tell]
Εχτισα το δικο μου blog χρησιμοποιοντας το hugo.

[myblog repository](https://github.com/StavroulaAnatolaki/myblog/tree/main)
[My blog](https://stavroulas-blog.netlify.app/)

**Comment by Alessenia** [comment]
Καλησπέρα! Καλό θα ήταν να προσθέσεις και κάποιο άρθρο,σχετικό με το μάθημα και την εισαγωγή σου, στο blog σου καθώς αυτός είναι και ο ρόλος ενός blog. Καλή συνέχεια!

**Reply by StavroulaAnatolaki** [reply]
Σε ευχαριστώ για τη συμβουλή σου.
